frontend/app/embed.ts

```diff
@@ -47,85 +48,98 @@ function createFrame({
 }
 
 function init() {
-  const node = document.getElementById(window.remark_config.node || NODE_ID);
+  window.REMARK42 = window.REMARK42 || {};
+  window.REMARK42.createInstance = createInstance;
 
-  if (!node) {
-    console.error("Remark42: Can't find root node.");
-    return;
-  }
+  if (window.remark_config) {
+    const remark42Instance = createInstance(window.remark_config);
 
-  try {
-    window.remark_config = window.remark_config || {};
-  } catch (e) {
-    console.error('Remark42: Config object is undefined.');
-    return;
+    if (remark42Instance) {
+      window.REMARK42.changeTheme = remark42Instance.changeTheme;
+      window.REMARK42.destroy = remark42Instance.destroy;
+    }
   }
 
-  if (!window.remark_config.site_id) {
-    console.error('Remark42: Site ID is undefined.');
-    return;
-  }
+  function createInstance(remark_config: CommentsConfig) {
+    let initDataAnimationTimeout: NodeJS.Timeout | null = null;
```

Comment:
should be `let initDataAnimationTimeout: number | null = null`; https://stackoverflow.com/a/50428377/13329651

Reply:
![Screen Shot 2563-06-24 at 18 30 29](https://user-images.githubusercontent.com/21255931/85548090-cb201780-b648-11ea-8303-d0c035cad183.png)